### PR TITLE
Nemo 7322/volunteer faq

### PR DESCRIPTION
**Pull request by sramalho94**
### Ticket(s)

_Required unless this is just a change to documentation._

- [7322](https://airtable.com/appfJZShN8K4tcWHU/pagXdnDYi0tVl4u8R?HjEAY=recNpeBLvszRcu7Yq)

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Description

As Part of the Task Group to improve the Get Involved Page, I've added a FAQ section. I modeled this FAQ section after the FAQ section in the Home page. Content for the FAQ section was taken from this [doc](https://docs.google.com/document/d/1qx6Wsh5LEXPb7X3MkntknGlfaQMveZObdkOkPagikhE/edit?usp=sharing).

### Checklist:

_Please delete any options that are not relevant_

- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings in the console
- [X] There are no new linting errors/problems in my code
- [X] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: content/get-involved.ts [hunk 1]
```diff
@@ -83,6 +84,48 @@ const getInvolvedContent: GetInvolvedContent = {
...
+    },
+    {
+      id: 'get-involved-faq6',
+      summary: 'Are there opportunities for advancement or leadership within the volunteer community?',
+      details: 'Definitely! We believe in nurturing talent and leadership. Every one of our Team Leads, Corporate Officers, and Board Members started out as an individual contributor volunteer, just like you might become. Volunteers are always encouraged to step up, utilize their unique skills, and contribute in a way that pushes our organizational mission and goals forward.',
+    },
+    {
+      id: 'get-involved-faq7',
+      summary: 'Who would I be reporting to or working under?',
+      details: 'As a volunteer, you\'ll be a part of a team. You\'ll report directly to the team leader/and or the individual who assigns tasks. We believe in a collaborative environment where everyone has a voice.',
```
Comment (e-papanicolas) [inline content/get-involved.ts:126]: This should be "...team leader and/or..."

File: pages/get-involved.tsx [hunk 2]
```diff
@@ -129,6 +130,19 @@ function Partner() {
   );
 }
 
+function GetInolvedFAQ() {
```
Comment (e-papanicolas) [inline pages/get-involved.tsx:133]: Can you please correct the spelling, missing a "v" 😄

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
Tiny changes, otherwise looks good. Glad we are adding this.

File: pages/get-involved.tsx (hunk 2 above)
Comment (sramalho94) [reply pages/get-involved.tsx:133]: @e-papanicolas lmaoo sorry about that. clearly my illiteracy has reached crippling levels

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**